### DISCUSSION TITLE
Vitest does not load .env file?

### DISCUSSION THREAD

**Discussion by IgnusG** [vitest-dev/vitest#768] [Q&A] [answered]
I'm not sure what I'm doing wrong as my config is note that complex:

```ts
/// <reference types="vite/client" />

import react from "@vitejs/plugin-react";
import { defineConfig } from "vite";
import tsconfigPaths from "vite-tsconfig-paths";

// https://vitejs.dev/config/
export default defineConfig({
  plugins: [react(), tsconfigPaths()],
   test: {
    globals: true, // Required for @testing-library
    include: ["**/*.test.{ts,tsx}"],
    environment: "jsdom",
    setupFiles: "./test/setup.ts",
  },
});
```

The env file is located at `.env` and has a single `VITE_...`  definition that's used through `import.meta.env.VITE...`.  
Do you guys have any tips on what could be the issue?

**Comment by sheremet-va** [comment, marked as answer]
I think this is a bug. I've tested it with stackltiz and can confirm env is not set: https://stackblitz.com/edit/vitest-dev-vitest-femj7g?file=test/basic.test.ts

You can use this as reproduction ^

**Reply by sheremet-va** [reply]
No need for an issue, I've created a PR to fix this: https://github.com/vitest-dev/vitest/pull/770
